overrides/Notifications/ui/NotificationAreaOverlay.ui

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
I think our JSON assets are parsed in a way that we allow for comments in them.

I'd appreciate if we could add why we need this override for Light and Shadow.

If I get it correctly, this is do move the notification area to the bottom left of the screen (instead of top right corner)?

```suggestion
/** Move the notification area to bottom-left corner of the screen to avoid overlaps with inventory. */
{
```